spring-cloud-dataflow-docs/src/main/asciidoc/configuration-kubernetes.adoc

```diff
@@ -37,7 +37,7 @@ These properties are also used when configuring the <<configuration-kubernetes-t
 |Namespace to use
 |environment variable `KUBERNETES_NAMESPACE`, otherwise `default`
 
-|nodeSelector
+|deployment.nodeSelector
```

Comment:
This had not been in the docs caused a bit of confusion in the community, so thank you for prefixing it.

We added global `nodeSelector` via https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/commit/82bebf397c48acfa21d265c1d94433a8176bf0c5s - shouldn't we also list the top-level property in the docs?

Reply:
We list the deployment properties in the table. For top-level deployer properties, we write separate sections under: https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#_application_and_server_properties